portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,50 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+    <body>
+
+    <!-- <div id="content">
+      <h1>Kiera Brown</h1>
+      <p>Hello World!</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
-    </div>
+    </div> -->
+            <div id="content">
+                <h1>Kiera Brown</h1>
+                <br>
+                <br>
+                <p>
+                    <a href="about.html">About Me</a>
+                    <br>
+                    <br>
+                    <a href="resume/Resume Brown, Kiera.pdf">Resume</a>
+                    <br>
+                    <br>
+                    <a href="https://github.com/kierabr1">Github</a>  
+                    <br>
+                    <br>
+                    <a href="https://www.linkedin.com/in/kierabr1/">Linkedin</a>
+                </p>
+
+
+
+
```

Comment:
Delete a few of the extra lines here. Leave one line if you need to

Reply:
done